xline/src/server/lease_server.rs

```diff
@@ -286,16 +287,18 @@ where
         request: tonic::Request<LeaseLeasesRequest>,
     ) -> Result<tonic::Response<LeaseLeasesResponse>, tonic::Status> {
         debug!("Receive LeaseLeasesRequest {:?}", request);
-        let leases = self
-            .lease_storage
-            .leases()
-            .into_iter()
-            .map(|lease| LeaseStatus { id: lease.id() })
-            .collect();
-        let res = LeaseLeasesResponse {
-            header: Some(self.lease_storage.gen_header()),
-            leases,
-        };
+
+        let is_fast_path = true;
```

Comment:
Why should we issue a proposal in the `lease_leases` method?

Reply:
Because some lease grant or revoke may not flush to db yet, so we need to let lease_leases go through the curp protocol.